AllReadyApp/Web-App/AllReady/Areas/Admin/Views/Event/Details.cshtml

```diff
@@ -168,11 +168,17 @@
             <div class="row">
                 <div class="col-md-12">
                     <h3>Requests</h3>
-                    &nbsp;
                 </div>
             </div>
 
             <div class="row">
+                <div class="col-md-12">
+                    <button type="button" asp-action="Create" asp-controller="Request" asp-area="Admin" asp-route-eventId="@Model.Id" class="btn btn-default">Create Request</button>
```

Comment:
The link tag helper doesn't target the button element. This needs to be an <a> tag. I have a fix incoming